src/apprt/embedded.zig

```diff
@@ -1346,13 +1347,16 @@ pub const CAPI = struct {
         opts: *const apprt.runtime.App.Options,
         config: *const Config,
     ) !*App {
-        var core_app = try CoreApp.create(global.alloc);
-        errdefer core_app.destroy();
+        var core_app = try global.alloc.create(CoreApp);
+        errdefer {
+            core_app.deinit();
+            global.alloc.destroy(core_app);
+        }
```

Comment:
I believe core_app needs to be inited after this line seems like it's initialization was removed

Reply:
Fixed.